sympy/physics/control/lti.py

```diff
@@ -378,27 +400,98 @@ def is_biproper(self):
 
 
 class Series(Basic):
+    """
+    A class for representing product of transfer functions (or ``Parallel`` objects).
+    All the transfer functions (or ``Parallel`` objects) should be using the same complex variable ``var`` of the
+    Laplace transform.
+
+    Examples
+    ========
+
+    >>> from sympy.abc import s, p, a, b
+    >>> from sympy.physics.control.lti import TransferFunction, Series, Parallel
+    >>> tf1 = TransferFunction(a*p**2 + b*s, s - p, s)
+    >>> tf2 = TransferFunction(s**3 - 2, s**4 + 5*s + 6, s)
+    >>> tf3 = TransferFunction(p**2, p + s, s)
+    >>> S1 = Series(tf1, tf2)
+    >>> S1
+    Series(TransferFunction(a*p**2 + b*s, -p + s, s), TransferFunction(s**3 - 2, s**4 + 5*s + 6, s))
+    >>> S1.var
+    s
+    >>> S2 = Series(tf2, Parallel(tf3, -tf1))
+    >>> S2
+    Series(TransferFunction(s**3 - 2, s**4 + 5*s + 6, s), Parallel(TransferFunction(p**2, p + s, s), TransferFunction(-a*p**2 - b*s, -p + s, s)))
+    >>> S2.var
+    s
```

Comment:
here, you need to provide more examples for exactly how Series is supposed to be used. You need to add more elaborate examples here. You should show tf1 * tf2 returns a Series object, most of the users will prob be creating Series objects using * instead of Series().

You also need to show what happens when a Series object is added/multiplied with other Series, Parallel, TF objects.

Reply:
This docstring will be the primary source for the users to find out how this class works, other than the member function docstrings. You need to help them as much as you can by explicitly showing how things work. 

Just think if you are a new user and have absolutely no idea how any of this work, how much will you know and will be able to do after reading the docs.

Reply:
I don't think the users will use `Series` or `Parallel` class directly. So, I have added some more examples in `TransferFunction` class level docstrings like how by using `+, -, *` with different transfer functions, we get Series and Parallel objects.

Reply:
Alright, fair enough